proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -46,6 +47,23 @@ message Repository {
   bool deprecated = 8;
   // deprecation_message is the message shown if the repository is deprecated.
   string deprecation_message = 9;
+  // owner_name is the name of the owner of the repository,
+  // either a username or organization name.
+  string owner_name = 10;
```

Comment:
Why do we need to start including the owner name here?

Reply:
this is kind of a preparation for repositories recommendation, where this is needed in order to link to the repository page by just the repository message.
we can exclude this from this PR but I think it makes sense to include it here while we are adding fields to repository, wdyt?

Reply:
I think we should avoid customizing the generic `buf.alpha.registry.v1alpha1.Repository` message with display fields that are meant for only certain usages of our frontend. Owner name seems still generically useful, but related to the comments below verification status of the owner or commit name seem more separate.

Reply:
discussed offline, we are going to remove verification status and last commit time from `Repository`, and might discuss on some endpoints that is more frontend specific later